profitbricks-rest/src/main/java/org/apache/jclouds/profitbricks/rest/domain/Metadata.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jclouds.profitbricks.rest.domain;
+
+import com.google.auto.value.AutoValue;
+import org.jclouds.json.SerializedNames;
+
+@AutoValue
+public abstract class Metadata {
+
+    public abstract String createdDate();
```

Comment:
These were date before, but test keep failing because of a parse error:

```
java.lang.IllegalArgumentException: Error parsing data at 0
	at java.text.DateFormat.parse(DateFormat.java:366)
	at org.jclouds.date.internal.SimpleDateFormatDateService.cDateParse(SimpleDateFormatDateService.java:80)
```